### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
- [DEMO LINK](https://kryvets-vadym.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by Khristina23042004 — APPROVED** [review]
Good job!

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
+    width: $cardWidth;
+    height: $cardHeight;
+
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 1024px) {
```
Comment (denys-danyliuk) [inline src/styles/_catalog.scss:24]:
```suggestion
@media (max-width: 1023px) {
```

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
+  .catalog {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (denys-danyliuk) [inline src/styles/_catalog.scss:30]:
```suggestion
@media (max-width: 767px) {
```

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
+  .catalog {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (max-width: 488px) {
```
Comment (denys-danyliuk) [inline src/styles/_catalog.scss:36]:
```suggestion
@media (max-width: 487px) {
```

File: src/styles/_catalog.scss (hunk 1 above)
Comment (denys-danyliuk) [reply src/styles/_catalog.scss:24]: Знімок екрана 2022-10-11 о 15 29 32

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**